database/pSQLdatabase.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-console */
+/* eslint-disable max-classes-per-file */
+const { Sequelize, Model, DataTypes } = require('sequelize');
+const dotenv = require('dotenv');
+
+dotenv.config();
+const sequelize = new Sequelize('postgres://127.0.0.1:5432/SDC');
+
+class Price extends Model { }
```

Comment:
curious - why choose extending the model over sequelize.define? Is there any significant difference, or just preference?